ci/generate-slack-notification.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+OLDER_GIT_VERSION=$1
+NEWEST_GIT_VERSION=$2
+MSG=$3
+COLOR=$4
+WRAP_SLACK=$5
+GITHUB_PROJECT=$6
+
+cat << EOF > notify.team.sh
```

Comment:
Why do we need a script to generate another script!? :exploding_head:

Reply:
This is because as a confirmation for the promote or flip, we expect the developer to manually run some commands.